docs/react-native/docs/android/permissions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Permissions
+description: Android device grants permission for notification by default
+next: /react-native/docs/android/progress-indicators
+previous: /react-native/docs/android/interaction
+---
+
+# Understanding permission
+
+On Android, notification permission is granted by default. However, user can revoke this permission through your app's settings or the notification itself. In addition,user can block notifiation at three levels; application-wide, channel groups and channels. 
+
+# Checking permission
+
+For each level, you can check whether notifcation permission is enabled on as-need basis or with event listeners.
+
+## Application wide
+
+To check whether the user has enabled application-wide notifications, call [`getNotificationSettings`](/reference/getnotificationsettings). The `authorizationStatus` attributes will returns `0` if user has disabled the permission, and `1` if it's granted.
+
+
+```js
+import notifee from '@notifee/react-native';
+
+async function checkNotificationPermission() {
+  const settings = await notifee.getNotificationSettings();
+
+  if (settings.authorizationStatus == 1) {
+    console.log('Notification permission has been granted');
+  } else if (settings.authorizationStatus == 0) {
+    console.log('Notification permission has been denied');
+  }
+}
+```
+
+In some cases where notification is disabled, you might want to give an option for user to open notification settings to enable it. After you have setup your UI, you can use Notifee's [`openNotificationSettings`](/reference/opennotificationsettings).
+
+## Channels & Channels Groups
+
+To check whether the user has enabled notification for specific channels or channel groups, call either [`getChannel`](/reference/getchannel) or [`getChannelGroups`](/reference/getchannelgroups). Both of these functions will return a respective object with `blocked` attributes in it.
+
+```js
+import notifee from '@notifee/react-native';
+
+async function checkChannelPermission() {
+  const channel = await notifee.getChannel();
+
+  if (channel.blocked) {
+    console.log('Channel notification has been disabled');
+  } else {
+    console.log('Channel notification has been enabled');
+  }
```

Comment:
If the attribute is called "blocked", we should really decide on two words to represent the two states and use them everywhere.

I consulted the APIs and they appear to use "enabled" and "blocked". So in all cases I would propose we use those words if possible - attribute names, API / method names, documentation here

Reply:
We will be using `enabled` for allowed state, and `blocked` for not-allowed state, correct?